bookmarklet.js

```diff
@@ -61,7 +61,7 @@ javascript:(
                                 if (errorMessage) {
                                     const errorType = (errorMessage.includes("Unexpected response status 400: The result is too big. Only queries with a maximum of 5000 results are supported.")) ? "ResultTooLargeError" : "InsightError";
                                     obj = {
-                                        title: "generated by https://github.com/ianmah/310-bookmarklet. Pls gib star",
+                                        title: `Generated by https://github.com/ianmah/310-bookmarklet. Pls gib star. ${errorMessage.trim()}`,
```

Comment:
Just to make it more concise, what about
```suggestion
                                        title: `${errorMessage.trim().replace("Unexpected response status 400: ", "")}. Generated by https://github.com/ianmah/310-bookmarklet. Pls gib star`,
```